test/integration/src/test/java/com/liferay/faces/portal/test/showcase/captcha/CaptchaGeneralTesterCompat.java

```diff
@@ -77,5 +81,64 @@ public void runCaptchaGeneralTest_C_Authenticated_NoLongerEnabled() throws Excep
 
 	}
 
+	/*
+	 * This test is to make sure that the fix for LPS-65011 does not regress.  It also tests the refresh button for the
+	 * SimpleCaptcha.
+	 */
+	@Test
+	public void runCaptchaGeneralTest_D_NonAuthenticated_IncorrectCaptchaAfterRefreshAndSubmit() throws Exception {
+
+		BrowserDriver browserDriver = getBrowserDriver();
+		WaitingAsserter waitingAsserter = getWaitingAsserter();
+		navigateToUseCase(browserDriver, "captcha", "general");
+
+		// 1. Enter the correct value into the *Captcha* field.
+		String correctCaptchaValue = getCorrectCaptchaValue(browserDriver);
+		browserDriver.clearElement(CAPTCHA_INPUT_XPATH);
+		browserDriver.sendKeysToElement(CAPTCHA_INPUT_XPATH, correctCaptchaValue);
+
+		// 2. Remember the first correct captcha value
+		String correctCaptchaValue1 = correctCaptchaValue;
+
+		// 3. Click on the *Submit* button and verify that an informational message is displayed indicating that the
+		// correct value was submitted.
+		browserDriver.clickElementAndWaitForRerender(submitButton1Xpath);
+		waitingAsserter.assertElementDisplayed(CAPTCHA_MSG_INFO_XPATH);
+		waitingAsserter.assertTextPresentInElement("You entered the correct text verification code",
+				CAPTCHA_MSG_INFO_XPATH);
+
+		// 4. Click the *Refresh CAPTCHA* button
+		waitingAsserter.assertElementDisplayed(REFRESH_CAPTCHA_XPATH);
+		browserDriver.clickElement(REFRESH_CAPTCHA_XPATH);
+		browserDriver.waitForElementEnabled(CAPTCHA_XPATH, true);
+
+		// 5. Enter the correct value into the *Captcha* field.
+		correctCaptchaValue = getCorrectCaptchaValue(browserDriver);
+		browserDriver.clearElement(CAPTCHA_INPUT_XPATH);
+		browserDriver.sendKeysToElement(CAPTCHA_INPUT_XPATH, correctCaptchaValue);
+
+		// 2. Remember the second correct captcha value
+		String correctCaptchaValue2 = correctCaptchaValue;
+
+		// 6. Click on the *Submit* button and verify that an informational message is displayed indicating that the
+		// correct value was submitted.
+		browserDriver.clickElementAndWaitForRerender(submitButton1Xpath);
+		waitingAsserter.assertElementDisplayed(CAPTCHA_MSG_INFO_XPATH);
+		waitingAsserter.assertTextPresentInElement("You entered the correct text verification code",
+				CAPTCHA_MSG_INFO_XPATH);
+
+		// 7. Get the correct captcha value for this third captcha rendered
+		correctCaptchaValue = getCorrectCaptchaValue(browserDriver);
+
+		// 8. Ensure that this final captcha value is not the same as the first
+		Assert.assertNotEquals("The final captcha value should not be the same as the first (" + correctCaptchaValue1 +
```

Comment:
@vsingleton FYI, forgot to mention this, but with all the refactoring work geared towards using the new Testing API, the convention moving forward is to make JUnit assertions with a Java-based textual comment that positively explains the purpose of the test, rather than an assertion message that explains the reason for failure. See code review commit.